### DISCUSSION TITLE
Set keybinding to open command mode with some string in it

### DISCUSSION THREAD

**Discussion by uwla** [qutebrowser/qutebrowser#6396] [Q&A] [answered]
Hello!

When press `O` it toggles command mode with the following string `open -t`. How can I do the same but for a custom string?

Like:

```
config.bind(',o', 'open {url:domain}' )
```

But I don't want the command to execute. I just want to go in command mode and have, for example, the string `open domain.example` in the command line, so that I can edit it.

**Comment by uwla** [comment]
I found a workaround but I'm not sure if this is the best option:

```
config.bind(',o', 'fake-key -g ":open {url:domain}"')
```

**Reply by bitraid** [reply]
Use [set-cmd-text](https://qutebrowser.org/doc/help/commands.html#set-cmd-text)

**Reply by uwla** [reply]
Thank you very much!

**Comment by The-Compiler** [comment, marked as answer]
Doing `:bind O` will show you what `O` is bound to (or `:bind` will show you all bindings):

> O is bound to 'set-cmd-text -s :open -t' in normal mode

**Reply by uwla** [reply]
Thanks! I didn't know that

I don't want to overwrite the defaults though, so I set to `,o`. My doubt was how to set text to command mode, and @bitraid pointed out that the proper way to do this is set-cmd-text.

Thanks for the quick answer

**Reply by The-Compiler** [reply]
I'm not saying you need to overwrite the default - just how you can find this kind of thing out in the future without having to ask and wait for an answer :slightly_smiling_face:
